src/language-js/printer-estree.js

```diff
@@ -5233,6 +5245,15 @@ function templateLiteralHasNewLines(template) {
   return template.quasis.some(quasi => quasi.value.raw.includes("\n"));
 }
 
+function isStyledComponentsCssTemplate(n, text, options) {
+  return (
+    n.type === "LogicalExpression" &&
+    n.right.type === "TaggedTemplateExpression" &&
+    n.right.tag.name === "css" &&
+    !privateUtil.hasNewline(text, options.locStart(n), { backwards: true })
```

Comment:
I copied it from `isTemplateOnItsOwnLine`, but honestly have no idea what it does. Is it something that should be there?

Reply:
I don't see why this is necessary. Did you try without this?